packages/client/hmi-client/src/services/charts.ts

```diff
@@ -746,8 +746,8 @@ export function createSuccessCriteriaChart(
 	options: BaseChartOptions
 ): any {
 	const targetState = `${targetVariable}_state`;
-	const data = riskResults[targetState]?.qoi || [];
-	const risk = riskResults[targetState]?.risk?.[0] || 0;
+	const data = riskResults?.[targetState]?.qoi || [];
+	const risk = riskResults?.[targetState]?.risk?.[0] || 0;
```

Comment:
Maybe we can add a FIXME here, it is rather odd that at the time we are rendering a chart that we need to check if the result we just got are there or not. CC @Tom-Szendrey

Reply:
sure Ill add the comment, I just noticed it was breaking here at some points and this fixed it

Reply:
I suspect this type of check is being used in the `createForecastChart` and maybe others as well 
Theyre all set up on computed values the same way if you create a ticket for the fixme you can throw @jryu01 or myself on it to check it out